public/js/searchList.js

```diff
@@ -30,28 +30,31 @@ const searchCardTemplate =
     `;
 
 /*
- * When the page loads, fetch initial organization data and render it
+ * When the page loads, fetches initial organization data and render it
  * in cards on the list.
  */
 window.onload = function() {
-  fetchOrganizations("all").then(organizations => {
-    createOrganizationCards(organizations);
-    createMarkers(organizations);
-  })
+  updateResults();
 }
 
 /*
- * Fetches the organization data with the given filters from the server.
- * @param {string} filterText The selected item category with which to filer results for.
+ * Requerys for organization data and refreshes page data.
  */
-function fetchOrganizations(filterText) {
-  return fetch("/discover", {
-    method: 'POST',
-    body: JSON.stringify({filter: filterText})
-  }).then(data => data.json());
+function updateResults() {
+  // Clear out current page data.
+  document.getElementById("search-list").innerHTML = "";
+  removeMarkers();
+
+  // Requery and repopulate page data.
+  const filter = document.getElementById("search-dropdown").value;
+  fetch("/discover/" + filter).then(data => data.json())
+  .then(organizations => {
+    createOrganizationCards(organizations);
```

Comment:
Nit: Make sure the second ```.then``` is an increased indentation.

This is more like personal preference but usually I think the best way to break up these kind of calls is something like

```
fetch(...)
    .then(() => foo())
    .then(() => bar())
    .thenCatch(() => {
      return baz;
    });
```

I think it makes the data flow slightly easier to follow.